inspirehep/modules/orcid/ext.py

```diff
@@ -40,15 +38,3 @@ def __init__(self, app=None, **kwargs):
     def init_app(self, app, **kwargs):
         """Initialize application object."""
         app.extensions['inspire-orcid'] = self
```

Comment:
Not sure tbh. Isn't this needed to create the extension (in order to declare the receivers etc.)? And also we are gonna need it for #1318. I can try unplugging the extensions although.